automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookTestUtils.scala

```diff
@@ -305,4 +308,35 @@ trait NotebookTestUtils extends LeonardoTestUtils {
       case other => fail(s"Unexpected kernel: $other")
     }
   }
+
+  def getLockedBy(workspaceBucketName: GcsBucketName, notebookName: GcsBlobName): IO[Option[String]] = {
+    google2StorageResource.use {
+      google2StorageDAO =>
+        for {
+          metadata <- google2StorageDAO.getObjectMetadata(workspaceBucketName, notebookName, None).compile.last
+          lockExpiresAt = metadata match {
+            case Some(GetMetadataResponse.Metadata(_, metadataMap)) if metadataMap.contains("lockExpiresAt") => Some(metadataMap("lockExpiresAt"))
+            case _ => None
+          }
+          currentlyLocked = lockExpiresAt match {
+            case Some(instantStr) => Instant.parse(instantStr).compareTo(Instant.now()) == 1
+            case None => false
+          }
+          lastLockedBy = if (currentlyLocked) {
+            metadata match {
+              case Some(GetMetadataResponse.Metadata(_, metadataMap)) if metadataMap.contains("lastLockedBy") => Some(metadataMap("lastLockedBy"))
```

Comment:
use `metadataMap.get("lastLockedBy")` instead of check `contain` and then use `apply`